### ISSUE TITLE
Date Import

### ISSUE THREAD

**Issue by Takaya213** [engram-design/FeedMe#146]
The date that gets passed by the RSS feed is e.g. Tue, 31 Jan 2017 15:26:42 +0000, but on import it imports at the time the task is run and not as the date above. I have selected it to be imported into the Post Date field. Any ideas as to why? Thanks.

**Comment by engram-design** [comment]
Not all dates can be automatically parsed and understood by Feed Me, due to the complexity and variations of different formats. Can you change the date in your feed? If unable to parse the date properly, it will default to the publish date

**Comment by Takaya213** [comment]
Ok, here is the debug output for the date entry, they all look pretty much the same, here are the first 2:
[postDate] => Craft\DateTime Object
        (
            [date] => 2017-01-31 15:26:42.000000
            [timezone_type] => 3
            [timezone] => Africa/Johannesburg
        )

  [postDate] => Craft\DateTime Object
        (
            [date] => 2017-01-26 20:22:20.000000
            [timezone_type] => 3
            [timezone] => Africa/Johannesburg
        )

**Comment by engram-design** [comment]
So that output looks correct? Note that there should be 3 date fields for an element - `dateCreated`, `dateUpdated` and `postDate`. Of all of them, only `postDate` can be modified, with the other two always matching the time of import.

**Comment by Takaya213** [comment]
Yes, both feeds are set to Add and Update. I am only pulling the date from the one feed. Could that be the issue? As I ran the first feed that I pull the date in with, and then the 2nd feed that I use to get an image related to the other feed, and I noticed that the post date comes in correctly before I run the 2nd feed where I don't pull in the date. As soon as the 2nd feed has finished and I check the entries the date changed to the time that it was run.
